proto/meta.proto

```diff
@@ -322,6 +323,23 @@ message RecoverRequest {}
 
 message RecoverResponse {}
 
+message AlterConnectorPropsRequest {
+  enum AlterConnectorPropsObject {
+    UNSPECIFIED = 0;
+    SOURCE = 1;
+    SINK = 2;
+    CONNECTION = 3;
+  }
+
+  uint32 object_id = 1;
+  map<string, string> changed_props = 2;
+  map<string, secret.SecretRef> changed_secret_refs = 3;
+  optional uint32 connector_conn_ref = 4;
```

Comment:
why we need to include the secret's connection in the request? The info is available in source/sink's catalog.

Reply:
Because this s is a new SecretRef from alter

Reply:
I dont get it. Are we allowing to change the CONNECTION used by existing SINK?